services/searchengine/bleveengine/search.go

```diff
@@ -17,6 +18,13 @@ import (
 const DeletePostsBatchSize = 500
 const DeleteFilesBatchSize = 500
 
+// Find the end of the term and include the wildcard if it exists
+// because we will put a wildcard at the end of the term regardless of whether a wildcard exists or not
+var wildcardRegExpForFileSearch = regexp.MustCompile(`\*?$`)
+
+// In excludedTerms case, we don't put a wildcard after the term.
+var exactPhraseRegExpForFileSearch = regexp.MustCompile(`"[^"]+"`)
+
```

Comment:
These comments can be modified to something more descriptive. Let's say:
```suggestion
// Regexp to match an asterisk at the end of the string. It is used
// in the file search logic to put a wildcard at the end of the search
// term regardless of whether a wildcard exists or not
var wildcardRegExpForFileSearch = regexp.MustCompile(`\*?$`)

// Regexp to mathch any quoted string. It is used in the file search
// logic to avoid adding a wildcard at the end of the quoted search
// terms, which need to be matched literally.
var exactPhraseRegExpForFileSearch = regexp.MustCompile(`"[^"]+"`)

```